inst/templates/ad_adcm.R

```diff
@@ -138,9 +138,9 @@ adcm <- adcm %>%
 
 # Save output ----
 
-dir <- file.path(getwd(), "tmp")
+dir <- tools::R_user_dir("admiral_templates_data", which = "cache")
 if (!file.exists(dir)) {
   # Create the folder
-  dir.create(dir)
+  dir.create(dir, recursive = TRUE, showWarnings = FALSE)
 }
```

Comment:
Could you add a comment that this code is needed for the admiral workflows only and that users should remove it?